.ci/azure/linux.yml

```diff
@@ -230,6 +231,25 @@ jobs:
     displayName: 'IE CAPITests'
     continueOnError: false
 
+  - script: |
+      wget -P tests/samples_smoke_tests/ https://storage.openvinotoolkit.org/repositories/openvino/ci_dependencies/test/samples_smoke_tests_data.zip     
+      python3 -m pip install  --upgrade pip
```

Comment:
remove upgrade pip and install pytest, setuptools - because it was done on the previous steps

Reply:
was deleted